src/backend/tasks_io/datafeeds/parsers/fms_api/fms_api_event_list_parser.py

```diff
@@ -59,6 +59,8 @@ class FMSAPIEventListParser(ParserJSON[Tuple[List[Event], List[District]]]):
         "darwin": ("dar", "Darwin"),
         "galileo": ("gal", "Galileo"),
         "hopper": ("hop", "Hopper"),
+        "johnson": ("joh", "Johnson"),
```

Comment:
Looks like FIRST reworked all the division codes? e.g. hopper is `hcmp` and "Archimedes presented by Kettering University" is `ARPKY` 🤮

Reply:
Yeah, I didn't know how we wanted to support this.
Is it a...
- "just change it, since FIRST changed it"
- "add some specific 2023 code"
- or something else?

Reply:
add the 2023 specific code. we can have multiple keys map to the same thing. e.g.

```
"hopper": ("hop", "Hopper"),
"hcmp": ("hop", "Hopper"),
```